src/common/adapters/AxelarAdapter.sol

```diff
@@ -19,17 +19,17 @@ import {IMessageHandler} from "src/common/interfaces/IMessageHandler.sol";
 contract AxelarAdapter is Auth, IAxelarAdapter {
     using CastLib for *;
 
-    IMessageHandler public immutable gateway;
+    IMessageHandler public immutable startpoint;
     IAxelarGateway public immutable axelarGateway;
     IAxelarGasService public immutable axelarGasService;
 
     mapping(string axelarId => AxelarSource) public sources;
     mapping(uint16 centrifugeId => AxelarDestination) public destinations;
 
-    constructor(IMessageHandler gateway_, address axelarGateway_, address axelarGasService_, address deployer)
+    constructor(IMessageHandler startpoint_, address axelarGateway_, address axelarGasService_, address deployer)
```

Comment:
Could we call it `entrypoint`? `startpoint` is not an oft used English term afaik.